### DISCUSSION TITLE
Welcome to novosga Discussions!

### DISCUSSION THREAD

**Discussion by rogeriolino** [novosga/novosga#359] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by fredcwbr** [comment]
Instalei a versao 2.0.8, e estou com alguns problemas em relacao ao websocket., 
Vi que na 2.1 foi alterado a forma de acesso .,
Pergunto, .como esta a 2.1? tem condições de produção?
